src/components/SummaryManagement/SummaryManagement.jsx

```diff
@@ -158,18 +151,33 @@ class SummaryManagement extends Component {
     this.setState({
       selectedSummaryGroupId: undefined,
       selectedSummaryGroupName: '',
-      //isEdit: false,
       summaryGroupStatusPopupOpen: false,
     });
   };
-  /*
-  onTeamMembersPopupShow = (teamId, teamName) => {
-    //this.props.getTeamMembers(teamId);
+
+  onTeamMembersPopupShow = async (summaryGroupId, name) => {
     this.setState({
       teamMembersPopupOpen: true,
-      selectedSummaryGroup: teamName,
-      selectedSummaryGroupId: teamId,
+      selectedSummaryGroup: name,
+      selectedSummaryGroupId: summaryGroupId,
     });
+    try {
```

Comment:
This looks like exactly the same functionality. You may only need one of them in your application unless there is a typo and these functions were meant to be different.

Reply:
update please check